integration-cli/docker_cli_run_test.go

```diff
@@ -2978,6 +2992,15 @@ func testReadOnlyFile(filename string, c *check.C) {
 	if !strings.Contains(string(out), expected) {
 		c.Fatalf("expected output from failure to contain %s but contains %s", expected, out)
 	}
+
+	out, err = exec.Command(dockerBinary, "run", "--read-only", "--privileged", "--rm", "busybox", "touch", filename).CombinedOutput()
+	if err == nil {
```

Comment:
Made the other changes, but not this because the rest of this method uses c.Fatalf()... even if we can't be consistent in the integration tests, I figure we should be consistent within a method.